docs/create/homebrew/index-homebrew.mdx

```diff
@@ -131,6 +148,118 @@ RedisTimeSeries supports downsampling with the following aggregations: avg, sum,
 ![time series compaction](images/testredis6.png)
 
 
+</TabItem>
+<TabItem value="redis-on-macos">
+
+There are two ways to install Redis on Mac OS:
+
+- [Installing Redis from source](/create/from-source/)
+- Using Homebrew
+
+Homebrew is the easiest and most flexible way to install Redis on MacOS. It is a package management software for MacOS.
+It automates Redis installation process, making it quick and easy to add Redis to your system. 
+
+Follow the below steps to install Redis on Mac OS using brew service:
+
+
+### Step 1: Install Homebrew
+
+Run the below command to install and start brew service:
+
+
+ ```bash
+  /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+ ```
+
+### Step 2: Install Redis using Homebrew package manager
+
+Follow the below command line to install Redis using brew service:
+ 
+ ```bash
+  brew update
+  brew install redis
+ ```
+
+
+### Step 3: Start Redis server
+
+
+Run the below command line to start the Redis database in the background:
+
+ ```bash
+  brew services start redis
+ ```
+
+In order to run the latest version of Redis, you will need to compil Redis from the source.
+[Follow this link](/create/from-source/) to learn more about it.
+
+
+### Step 4: Test if Redis server is running.
+
+ ```bash
+  redis-cli ping
+ ```
+
+It should return PONG. This command is often used to test if a connection is still alive.
+
+### Step 5: Launch Redis on system boot
+
+
+ ```bash
+  ln -sfv /usr/local/opt/redis/*.plist ~/Library/LaunchAgents
+ ```
+
+### Start Redis server via “launchctl” command
+
+```bash
+ launchctl load ~/Library/LaunchAgents/homebrew.mxcl.redis.plist
+```
+
+### Step 6: Run Redis service using a redis configuration file 
+
+ ```bash
+  redis-server /usr/local/etc/redis.conf
+ ```
+
+### Step 7: Interacting with Redis Client
+
+ ```bash
+  redis-cli
+  redis> set foo bar
+  OK
+  redis> get foo
+  "bar"
+ ``` 
+
+Please note: By default, Redis has 0-15 indexes for databases, you can change that number databases NUMBER in redis.conf.
```

Comment:
Suggest we remove this as using `SELECT` and the numbered databases is considered an antipattern and we shouldn't encourage people to use it.
```suggestion
```